modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/continuous/CacheContinuousQueryHandler.java

```diff
@@ -91,13 +92,32 @@
     /** */
     private static final long serialVersionUID = 0L;
 
+    /** @see #IGNITE_CONTINUOUS_QUERY_BACKUP_ACK_THRESHOLD */
+    public static final int DFLT_CONTINUOUS_QUERY_BACKUP_ACK_THRESHOLD = 100;
+
+    /** @see #IGNITE_CONTINUOUS_QUERY_LISTENER_MAX_BUFFER_SIZE */
+    public static final int DFLT_CONTINUOUS_QUERY_LISTENER_MAX_BUFFER_SIZE = 10_000;
+
+    /** */
+    @SystemProperty(value = "Size of the buffer of events to send acknowledge to backup", type = Long.class,
+        defaults = "" + DFLT_CONTINUOUS_QUERY_BACKUP_ACK_THRESHOLD)
+    public static final String IGNITE_CONTINUOUS_QUERY_BACKUP_ACK_THRESHOLD = "IGNITE_CONTINUOUS_QUERY_BACKUP_ACK_THRESHOLD";
+
+    /** */
+    @SystemProperty(value = "Maximum size of continuous query listener buffer. " +
```

Comment:
The maximum size of the continuous query listener buffer. 10% of events are dropped once the buffer is full.